Vagrantfile

```diff
@@ -53,7 +53,8 @@ Vagrant.configure(2) do |config|
   config.vm.network :private_network, ip: _conf['ip']
 
   config.vm.synced_folder _conf['synced_folder'],
-      _conf['document_root'], :create => "true", :mount_options => ['dmode=755', 'fmode=644']
+      _conf['document_root'], :create => "true", :mount_options => ['dmode=755', 'fmode=644'],
+      SharedFoldersEnableSymlinksCreate: false
```

Comment:
Great work. Thanks @lzivadinovic. I have a small problem tho; when I run the provisioning the first time i get a message about an error regarding symlinking when phpmyadmin is being installed. Has this something to do with this? Second time i run the provisioning it goes well. Maybe It is just not ready to install it at the first run by coincident:

```TASK [Move phpmyadmin to webroot] **********************************************
fatal: [vccw.test]: FAILED! => {"changed": true, "cmd": "if [ ! -d \"/var/www/html/phpmyadmin\" ]; then mv /usr/share/phpmyadmin/ /var/www/html/; fi", "delta": "0:00:04.430178", "end": "2018-11-21 17:16:39.091696", "msg": "non-zero return code", "rc": 1, "start": "2018-11-21 17:16:34.661518", "stderr": "mv: cannot create symbolic link '/var/www/html/phpmyadmin/doc/html': Read-only file system", "stderr_lines": ["mv: cannot create symbolic link '/var/www/html/phpmyadmin/doc/html': Read-only file system"], "stdout": "", "stdout_lines": []}```

Reply:
@adionditsak Yeah, this was problem. Removing shared folder enable symlink allows me to boot it from first try.